docs/source/user-guide/mission-modeler-guide/activity-types/effect-model.md

```diff
@@ -0,0 +1,128 @@
+# Effect Model
+Every activity type has an associated "effect model" that describes how that activity impacts mission resources. An effect model is a method on the activity type class annotated with `@EffectModel`; there can only be one such method, and by convention it is named `run` that accepts the top-level `Mission` model as a parameter. This method is invoked when the activity begins, paused when the activity waits a period of time, and resumes when that period of time passes. The activity's computed duration will be measured from the instant this method is entered, and extends either until the method returns or until all spawned children of the activity have completed -- whichever is longer.
+
+**Exceptions:**
```

Comment:
Minor Suggestion: Looking at this being bolded on its own line, is this supposed to be a header? A similar question for **Actions:** on line 7.

Reply:
Ahh 100%, just fixed this to be `## Effect Model` and made the subsections `### *`.

Reply:
Effect Model should stay as `# Effect Model`, since it's the title of the document, and the second levels headers should stay as `##`. What I meant is this line renders as:

> **Exceptions:** If an uncaught exception is thrown from an activity's effect model, the simulation will halt. No later activities will be performed...

But based on the structure of the document, it *looks* like it was intended to render as:

> **Exceptions:** 
> 
> If an uncaught exception is thrown from an activity's effect model, the simulation will halt. No later activities will be performed...

Which is why I was asking if `**Exceptions:**` was meant to be a header.

Reply:
Ohh I see! Just changed it to `# Effect Model` and made those `** ... **` items true subsections.
Thanks!